gnuboardProject/backend/src/auth/auth.controller.ts

```
@@ -1,8 +1,12 @@
-import { Controller, Post, Body, UnauthorizedException, ConflictException, UseGuards, Request, Get } from '@nestjs/common';
+import { Controller, Post, Body, UnauthorizedException, ConflictException, UseGuards, Request, Get, Query, BadRequestException } from '@nestjs/common';
 import { AuthService } from './auth.service';
 import { UsersService } from '../users/users.service';
 import { LocalAuthGuard } from './local-auth.guard';
 import { JwtAuthGuard } from './jwt-auth.guard';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { UploadedFile, UseInterceptors } from '@nestjs/common';
+import { diskStorage } from 'multer';
+import { extname } from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)